Xext/shm.c

```diff
@@ -97,6 +97,7 @@ in this Software without prior written authorization from The Open Group.
 
 
 typedef struct _ShmScrPrivateRec {
+    Bool initialized;
```

Comment:
Do we need this flag at all? It looks like it’s set, but not really used. Or it’s intended for external usage?

Reply:
Maybe print some warning (or a fatal error?) if a repeated init or deinit is attempted? To add some meaning to that flag 🙂